### ISSUE TITLE
Jquery is not working in the simple html page

### ISSUE THREAD

**Issue by jaylu2204** [webpack/webpack-dev-server#1420]
Here this is the simple html registration page. My form is in one division. When we will click on start button at that time my form division need to hide and new chatpage division need to visible. I'm making chat website using node js , socket.io and jquery. Right now I'm using that html page using node js server. Everythings going perfect but when I pressed start button, my new division is not displaying as well old division is not hiding.

![edit3](https://user-images.githubusercontent.com/24907749/41288163-de5f4ad6-6e34-11e8-9e28-1829a778ef1f.jpg)

<!DOCTYPE html>
<html>
<head>
    <title>Telepathy</title>
    <link rel="stylesheet" type="text/css" href="mystyle.css">

    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
    <script src="/socket.io/socket.io.js"></script>

    <script>
        // socket = io("http://localhost:11000/");

        $(document).ready(function(){
            $('#b1').click(function(){
                console.log("Inside.");
                $('#regi').hide();
                $('#btn_class').hide();
                $('#chatpage').show();
            });
        });

        function setUsername() {
            console.log("Inside the function.");    
            };
    </script>
</head>

<body background="edit3.jpg" style="margin: 0px; background-repeat: no-repeat;">
    <div id="regi" class="regi">
        <form action="" id="Registration" method="">
            <center>
                <table cellpadding="1" cellspacing="2" id="shadow1">
                    <tr>
                        <td colspan="3"><h1 class="regi_name" style="font-family: TheBlacklist; padding-bottom: 3.5%;font-size: 55px; color: black;"><u>User Details</u></h1></td>
                    </tr>

                    <tr>
                        <td style="padding-top: 2%; padding-right: 25%; padding-bottom: 6%;"><input type="text" name="fname" id="fname" placeholder="Enter First Name"></td>

                        <td rowspan="2"></td>

                        <td style="padding-top: 2%; padding-right: 25%; padding-bottom: 6%;"><input type="text" name="lname" id="lname" placeholder="Enter Last Name"></td>
                    </tr>

                    <tr>
                        <td style="padding-top: 2%; padding-right: 25%; padding-bottom: 6%;"><input type="email" name="email" id="email" placeholder="Enter MailID"/></td>

                        <td style="padding-top: 2%; padding-right: 25%; padding-bottom: 6%;"><input type="text" name="age" id="age" placeholder="Enter Age"/></td>
                    </tr>

                    <tr>
                        <td style="padding-top: 2%; padding-right: 25%; padding-bottom: 8%;">
                            <select name="city" id="city">
                                <optgroup label="Country"></optgroup>
                                <option value="uk">UK</option>
                                <option value="usa">USA</option>
                                <option value="india">India</option>
                                <option value="russia">Russia</option>
                            </select>
                        </td>

                        <td><button id="b1" class="input_btn" onclick="setUsername()" style="padding-top: 2%; padding-bottom: 8%;">Start</button></td>

                        <td style="padding-top: 2%; padding-right: 25%; padding-bottom: 8%;">
                            <select name="gender" id="gender">
                                <optgroup label="Gender"></optgroup>
                                <option value="male">Male</option>
                                <option value="Female">Female</option>
                            </select>
                        </td>
                    </tr>
                </table>
            </center>
        </form>
    </div>

    <div id="chatpage" style="width: 20%; height: 80%; border: 2px solid black; display: none;">
        <h2>Peoples</h2><hr>
    </div>
</body>
</html>

**Comment by evilebottnawi** [comment]
It looks like you just deleted our lovely crafted issue template. It was there for good reasons. Please help us solving your issue by answering the questions asked in this template. I'm closing this. Please either update the issue with the template and reopen, or open a new issue.
